pyproject.toml

```diff
@@ -41,12 +41,12 @@ keywords = [
   'labstreaminglayer',
   'LSL',
   'neuroimaging',
+  'neurophysiology',
   'neuroscience',
   'python',
   'real-time',
 ]
-license = 'BSD-3-Clause'
-license-files = ['LICENSE']
+license = {file = 'LICENSE'}
```

Comment:
Also fixing the license file warning you got, the definition was not compliant with [PEP-621](https://peps.python.org/pep-0621/#license).